extract-meta

```diff
@@ -39,7 +39,7 @@ function parseFile(filepath) {
     const urlpath = filepath.split(path.sep).join('/');
     let src;
 
-    if (!['.jsx', '.js'].includes(path.extname(filepath))) {
+    if (!['.js'].includes(path.extname(filepath))) {
```

Comment:
We currently do not have any `.jsx` files in our source code, so I figured they could be used in a way to denote components that _should not be picked up by extract-meta_, so that we can use them internally in JS. It's an easy solution to have the spinners, in this case, in separate files and not in one huge `Loading.react.js` file.